features/storage/TESTS/blockdevice/default_block_device/basic/basic.cpp

```diff
@@ -0,0 +1,195 @@
+/*
+ * mbed Microcontroller Library
+ * Copyright (c) 2006-2016 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+/* The following copyright notice is reproduced from the glibc project
+ * REF_LICENCE_GLIBC
```

Comment:
This might be problematic, please revise this file and this license

Reply:
@OPpuolitaival  license check is green, fails to find this

Reply:
> @OPpuolitaival license check is green, fails to find this

I can see why, we look for `gnu general public` but this is `gnu library general public`.

Reply:
Fixes the license check https://github.com/ARMmbed/mbed-os/pull/7911 to cover this

Reply:
Tests with GNU license has been removed